.stylelintignore

```diff
@@ -0,0 +1,3 @@
+/dist
+/storybook
+/coverage

```

Comment:
no need to add this ignore here the directory should only be created by automation tools

Reply:
Automation tools or running locally, otherwise we'll have to remember to manually delete it